manual/manual/cmds/intf-c.etex

```diff
@@ -297,7 +297,7 @@ then loads dynamically these libraries, and resolves references to the
 required primitives, before executing the bytecode.
 
 This facility is currently supported and known to work well under
-Linux, MacOS~X, and Windows.  It is supported, but not
+Linux, MacOS, and Windows.  It is supported, but not
 fully tested yet, under FreeBSD, Tru64, Solaris and Irix.  It is not
```

Comment:
While we are dusting our OS lists, is it useful to mention Tru64 and Irix as supported when they have been discontinued (without successor if I am not mistaken)?

Reply:
You are correct.  Actually this whole paragraph was outdated, since dynamic linking of C/OCaml stub code is supported everywhere except Cygwin 64 (for the moment).  So I updated the whole paragraph to say so.